### PR TITLE
[ID] Update osu!academy

### PR DESCRIPTION
**Pull request by febrimuhnas**
add needs_cleanup tag

### CODE REVIEW

File: wiki/osu!academy/id.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
+---
+needs_cleanup: true
```
Comment (MegaApplePi) [inline wiki/osu!academy/id.md]: @cl8n do translations also get this too?

Comment (cl8n) [reply wiki/osu!academy/id.md]: content/meaning needs work => add to en
the translation itself needs work, but en is fine => add here

Comment (MegaApplePi) [reply wiki/osu!academy/id.md]: the translation copied en, but en needs work => add to en only?

Comment (cl8n) [reply wiki/osu!academy/id.md]: ya

**Pull request merged**